### ISSUE TITLE
Cannot find name ‘HockeyApp’

### ISSUE THREAD

**Issue by ChrisK1994** [hicsail/ionic-hockeyapp#9]
When i try to inject this module into `app.component.ts` constructor I get error, why?

`import { HockeyApp } from 'ionic-hockeyapp';`

```
constructor(public platform: Platform,
    public app: App,
    public hockeyApp: HockeyApp) { }

```
and I have imported HockeyApp in `app.module.ts` and made it a provider

**Comment by gregfrasco** [comment]
What current version is installed?

**Comment by ChrisK1994** [comment]
HockeyApp: 0.1.6
Ionic CLI: 3.10.3
Cordova CLI: 7.0.1
Ionic-Angular: 3.6.1

**Comment by gregfrasco** [comment]
Fixed in v0.1.7
